### DISCUSSION TITLE
Questions over how to address odd Hyjal progressive patch changes

### DISCUSSION THREAD

**Discussion by Cooties-cc** [chromiecraft/chromiecraft#7017] [PTR Feedback]
Not 100% sure PTR feedback was right topic, but figured was close enough. 
This is regarding what to do for Hyjal when addressing the issue of it being in the game since TBC launch, but not getting any patch adjustments or changes till post Black Temple patch (2.1). This was due to the complete trash attunement system they had for it, before it was modified much later around 2.1. I wasn't sure if we should write up tickets about each problem, so instead asking here made more sense. 

I combed through all patches from TBC release up to 3.3.5a ending, trying to get the core issues changed. I might have missed some, so it's not bullet proof perfect. If no patch was listed or mentioned, then nothing happened with Hyjal (or it was very minor thing). This is the following things I found that related to the raid.

Patch 2.1
- All 25 man raid bosses who drop set tokens will now drop an additional token (from 1 to 2)
- A new set of more powerful gems can be found by raiding Hyjal and Black Temple. Their stats exceed those of previous gems that could be found (intro of epic gems + cuts)

Patch 2.1.2
- The number of preliminary waves before each boss in the Caverns of Time: Battle for Mount Hyjal event has been reduced from 12 to 8, and some of the larger waves have fewer creatures than before

Patch 2.1.3
- Fixed an issue with the movement of Archimonde's Doomfire

Patch 2.2
- Faction (rep I assume) and loot rewards have been added to non-boss mobs in the Battle of Mount Hyjal raid event. The faction [rep] awarded by bosses has been decreased to balance this
- The health of Jaina and Thrall have been increased again from their temporarily lowered values. They should be able to absorb quite a bit more incidental damage during the event
- Archimonde has received several adjustments and fixes to prevent some of the harder to counter combinations of his abilities. Archimonde's Soul Charge silence has been lowered in duration to 4 seconds, and the death of the Priest with Spirit of Redemption will no longer create two Soul Charges. The casting time of Archimonde's fear has been increased to 1.5 seconds, and he should now use his Fear ability on a much more consistent interval. Archimonde will also no longer cast Grip of the Legion on very far away targets
- Archimonde will no longer cast Grip of the Legion on targets that have just been hit with Air Burst
- Crypt Fiends around Azeroth and at Hyjal Summit are now more totem friendly

Patch 2.3
- Increased the reputation awarded for killing Archimonde in Hyjal Summit

Patch 2.4
- All 25-player raid bosses have had their cash drops increased
- All 25-player raid bosses that drop set tokens will now drop an additional token! (from 2 to 3)
- Badges of Justice have been added to all raid bosses who did not previously have them
- Players will no longer require an attunement quest to enter Hyjal. The Vials of Eternity quest is no longer necessary for entry to Hyjal, however the quest is still required for faction rings
- Scale of the Sands reputation will now be awarded in Hyjal at a much higher rate
- Azgalor's Rain of Fire now affects a smaller area
-  Archimonde's Grip of the Legion will no longer target pets
- The area around Archimonde in Battle for Mount Hyjal has been decreased in size and players are no longer able to go into the hills around his area. Also, the hitbox of Archimonde has been reduced significantly

Patch 3.1.2
- Players can no longer become saved to an instance when they aren't in the instance during a boss kill

To summarize this, since all of these things happened _post_ Black Temple, and Hyjal is releasing _before_ that, it would "technically" mean : 

Until Black Temple release:
- Bosses should only drop 1 token (2 on BT release, 3 on MgT/SWP release)
- Epic gems not avail 
- Waves set to 12, not 8, and add more mobs to large waves
- Remove loot and rep from normal mobs
- Lower HP of Jaina & Thrall
- Not sure how much of the 2.2 archi changes can be done
- Legion Grip can be cast on Air Burst targets

Until ZA release
- Lower Archimonde’s rep rewarded on kill

Until SWP (or MgT?) release
- No BoJ’s on bosses
- Nerf rep from trash (and fix 1st wave farming?)
- Increase Azgalor’s RoF area of effect
- Allow Legion Grip to target pets
- Increase Archimonde’s hitbox size (cant do anything about map size)

I wasn't sure how much of this would actually stick, nor even "correct" simply wouldn't be included due to pushback from community / not possible code wise. If going for a post 2.4 or "wotlk version" build, then a lot of this is removed. However, I know on a ticket of mine (reduce 3 tokens to 2) that was confirmed correct, isn't till a much later date and was a pre 2.4 change. It was also like this on tbcc, however they released the content as a package, whereas ours is stagnated. 

I guess ultimately I wanted to give a overview on changes for the raid, and wanted feedback on ones that _aren't_ going to happen. That way tickets can be made for ones that still need to change.

**Comment by balleny** [comment]
many of the things are "core" topics or mechanics. hence, they wont implement 12 waves.

loot topics might be worth discussing, but people are already crying about 2 instead of 3 tokens and now imagine 1 token to drop only.
reputation would be an interesting point too - be it the decrease or the total shift to bosses only. effort wise it would also be low because there are literally 5 different mob types in hyjal.

the gems from the ores again is more a script thing because of the spawn being linked to the script events and is unlikely to be changed.

but, because practically all DEVs are currently burned out, busy IRL or simply not available, there wont be much of a discussion  about changing things from wotlk defaults

**Reply by Cooties-cc** [reply]
Sounds about more or less what I thought about the "issues". I guess taking it from that angle, we could remove the following from being a "ticketable problem" : Token counts, epic gems + pats, wave counts, trash loot, and probably none of the archi pre-2.2 changes. 

From that, possible topics to report a ticket about to see what staff aligns with would be for the prog mod : Rep on trash and to what amount based on patches, lower hp tunes on jaina+thrall, allow grip of the legion on air burst targets, lowering rep from archi kills,  increase azgalor RoF radius, allow grip of the legion on pets, and increase the hitbox size of archi. These would be based out of the prog mod since they are changed as content rolls out, but are feasible changes. 

And lastly issues listed that are probably ignored since they are baseline already were : BoJ on bosses, the part about totems and crypt fiends, and people getting raid ID saved when outside the raid. These most likely aren't a problem since BoJ removal has been a baseline change for a while, and the other 2 _probably_ are a more core client change issue, not Hyjal itself. 

Thanks for the feedback. I'll try and sort out some tickets and see where they go with that.
